src/main/java/frc/robot/PowerController.java

```diff
@@ -0,0 +1,168 @@
+package frc.robot;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import edu.wpi.first.wpilibj.PowerDistribution;
+import frc.robot.subsystems.tank.TankSubsystem;
+import static frc.robot.Constants.TankConstants.*;
+
+public class PowerController {
+
+    private static PowerDistribution PDP = new PowerDistribution();
+
+    private ControllableSubsystem tankSubsystem;
+
+//    private ControllableSubsystem climbSubsystem;
+
+   // private ControllableSubsystem intakeSubsystem;
+
+   // private ControllableSubsystem shooterSubsystem;
+
+    private int currentCurrentLimit;
+
+    private int totalSustainableCurrent;
+
+    //TODO find current max value
+
+
+    private List<ControllableSubsystem> subsystems;
+
+    public PowerController(ControllableSubsystem tankSubsystem) {
+        subsystems = new ArrayList<ControllableSubsystem>();
+
+        subsystems.add(tankSubsystem);
+
+        this.tankSubsystem = tankSubsystem;
+
+        currentCurrentLimit = 350;
```

Comment:
1. Could these be constants, or will they change during a match?
2. Are you sure you want to do integer arithmetic with these?